core/modules/dashboard/css/dashboard.css

```diff
@@ -0,0 +1,10 @@
+/**
+ * @file
+ * Dashboard CSS.
+ */
+
+ /* Align the manage links to the right */
+ .dashboard-blocks .admin-page {
+   text-align: right;
+   font-size: .85em;
```

Comment:
This font size makes links too small compared to the rest of the similar links throughout the admin UI.

Reply:
Can you point me to a page with "similar links throughout the admin UI." for comparison?

Reply:
Table text is `font-size: 0.923em;` so I've used that. Slightly larger.

Reply:
Well, links in `/admin/config` are 16px. Anyway, 0.923em is much better now. Thanks 👍

Reply:
...the "show raw weights" link in `/admin/structure/menu/manage/user-menu`, `/admin/config/regional/language` is 0.9em FYI.